main.go

```diff
@@ -94,8 +103,16 @@ func main() {
 		dief("looking for test data: %v\n", err)
 	}
 	for _, name := range remove {
-		if err := os.RemoveAll(name); err != nil {
-			dief("removing %v: %v\n", name, err)
+		err := os.RemoveAll(name);
+		if err != nil {
+			failedToRemove[name] = err
+		}
+	}
+	
+	if len(failedToRemove) > 0 {
+		fmt.Fprintf(os.Stderr, "\n# Failed to remove below item, please remove manually\n")
+		for name, err := range failedToRemove {
```

Comment:
ideally we shouldn`t come to this line. however the delete attempt failed on ".git" folder when running on windows, complaining access denied (which is odd to me). 

agree that we should try harder to delete those file. 
but i think operation should continue if encounter deletion failure.